prow/cmd/deck/main.go

```diff
@@ -111,6 +119,27 @@ func handleData(ja *JobAgent) http.HandlerFunc {
 	}
 }
 
+func handleTide(ta *tideAgent) http.HandlerFunc {
```

Comment:
why not `func (ta *tideAgent) ServeHTTP(w http.ResponseWriter, r *http.Request)` ?

Reply:
That would probably work too.